src/components/deletePostModal/DeletePostModal.styled.tsx

```diff
@@ -9,8 +13,165 @@ export const AbsoluteDeleteWrapper = styled.div`
   top: 50%;
   left: 50%;
   transform: translateX(-50%);
-  background-color: white;
+  color: white;
+  background-color: rgba(35, 35, 35 ,1);
   width: 300px;
-  height: 300px;
+  height: 350px;
   z-index: 1;
-`;
\ No newline at end of file
+  padding: 20px;
+  border-radius: 5px;
+  `;
+
+export const DeleteModalHeader = styled.div`
+  width: 100%;
+  height: 40px;
+  margin: 0 0 15px 0;
+  background: rgba(40, 40, 40, 1);
+  border-bottom: rgba(45, 45, 45, 1);
+  
+  > .title {
+    font-size: 1rem;
+    padding: 12px 0 0 10px;
+  } 
+  `
+
+export const ConfirmForm = styled.form`
+  padding: 25px 0;
+  font-size: 0.8rem;
+
+  > .title {
+    padding: 8px 0 20px 0;
+    font-size: 0.9rem;
+  }
+`
+
+export const InputWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+`
+
+export const Input = styled.input`
+  &[type='text'] {
+    flex: 1 0;
+    min-width: 50px;
+    min-height: 25px;
+    font-size: inherit;
+    font-weight: 500;
+    padding-left: 5px;
+    background: rgba(41, 19, 21, 1);
+    padding: 12px 20px;
+    margin: 8px 0;
+    box-sizing: border-box;
+    color: rgba(199, 81, 86, 1);
+    border: 1px solid rgba(106, 32, 36, 1);
+    border-radius: 5px;
+    -webkit-transition: 0.5s;
+    transition: 0.5s;
+    outline: none;
+  
+  }
+  ::placeholder {
+    color: rgba(199, 81, 86, 1);
+    opacity: 0.5;
+    padding: 10px 0;
+    font-size: 0.8rem;
+    background-color: transparent;
+  }
+  &:link {
+    background: rgba(41, 19, 21, 1);
+  }
+  &:visited {
+    background: rgba(41, 19, 21, 1);
+  }
+  &:hover {
+    background: rgba(41, 19, 21, 1);
+  }
+  &:focus {
+    outline: none;
+    background-color: none;
+  }
+  &:active {
+    background: rgba(41, 19, 21, 1);
+  }
+`
+
+export const DeleteButton = styled.button`
+  width: 100%;
+  cursor: pointer;
+  font-size: 0.9rem;
+  font-weight: 500;
+  padding: 12px 50px;
+  margin: 25px 0px;
+  border: 1px solid rgba(106, 32, 36, 1);
+  border-radius: 5px;
+  color: rgba(199, 81, 86, 1);
+  background: rgba(41, 19, 21, 1);
+  transition: all 0.25s ease;
```

Comment:
Here for example, we want to replace `all` with what it is we want to transition.

Reply:
Here I'm setting the transition to the opacity property for the hover state. For now this how I could achieved the result.

Reply:
Fixed it. the transition I had was for the background instead of the opacity.